src/test/java/com/redhat/exhort/providers/Gradle_Provider_Test.java

```diff
@@ -46,13 +46,12 @@ abstract class Gradle_Provider_Test extends ExhortTest {
   // - build.gradle: the target manifest for testing
   // - expected_sbom.json: the SBOM expected to be provided
   static Stream<String> testFolders() {
-    return Stream.of(
-        "deps_with_ignore_full_specification",
-        "deps_with_ignore_named_params",
-        "deps_with_ignore_notations",
-        "deps_with_no_ignore_common_paths",
-        "deps_with_duplicate_no_version",
-        "deps_with_duplicate_different_versions");
+    return Stream.of("deps_with_ignore_full_specification");
```

Comment:
why are they commented out? maybe a leftover of the testing?

Reply:
My apologies, this is my mistake. forgot to add them back after DEBUG